welkin/models/calendar.py

```diff
@@ -117,3 +127,67 @@ def get(
         return super().get(
             f"{self._client.instance}/calendar/{route}", params=params, *args, **kwargs
         )
+
+
+class WorkHours(Resource):
+    @property
+    def id(self, value=None):
+        if not hasattr(self, "details"):
+            self.details = [{}]
+
+        self.details[0]["workHoursId"] = value
+
+    def create(self, repeating=True):
+        self.startDateTime = datetime.now(tz=timezone.utc)
+        if repeating:
+            self.endDateTime = self.startDateTime.replace(
+                year=self.startDateTime.year + 50
+            )
+
+        return super().post(f"{self._client.instance}/calendar/work-hours")
+
+    def get_all(
+        self,
+        from_date: datetime,
+        to_date: datetime,
+    ):
+
+        params = {
+            "from": from_date,
+            "to": to_date,
+        }
+
+        response = self._client.get(
+            f"{self._client.instance}/calendar/work-hours/", params=params)
+        
+        dict_version = {index: wh for index, wh in enumerate(response[0:-1])}
+        super().update(dict_version)
+        return self
+
+
+    def update(self, **kwargs):
+        return super().put(
+            f"{self._client.instance}/calendar/work-hours/{self.id}", kwargs
+        )
+
+
+class WorkerHours(Collection):
+    resource = WorkHours
+
+    def get(
+        self,
+        from_date: datetime,
+        to_date: datetime,
+        ids: list = None,
+        timezone: str = None,
+    ):
```

Comment:
this get should accept args and kwargs as well, this is for things like specifying pagination